lib/hydra/works/models/concerns/block_child_objects.rb

```diff
@@ -0,0 +1,21 @@
+module Hydra::Works
+  # Do not allow aggregation of child objects
+  module BlockChildObjects
+
+    def child_objects= objects
+      raise NoMethodError, "undefined method `child_objects=' for #{self}"
+    end
+
+    def child_objects
+      raise NoMethodError, "undefined method `child_objects' for #{self}"
+    end
+
+    def respond_to? method
```

Comment:
I don't think this is a good idea.  Why do we need to lie about what the object actually responds to?